CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing Guide
+
+Welcome to the Viam TypeScript SDK. Please read this guide prior to opening a pull request.
+
+## Development setup
+
+Developing on the TypeScript SDK requires macOS, Linux, or WSL / Bash for Windows. To get started, clone this repository:
```

Comment:
Makefile uses `rm -rf`, which will wreck havoc on a Windows machine if you try to use `cmd.exe` or PowerShell

Reply:
hm, I'll see if we can stop relying on that. We might not need to remove `dist` since it since it gets cleared by vite. we do need to clear out generated protobufs tho.

Reply:
I think getting rid of `rm -rf` is a good goal, but probably not the highest priority. You're right that it isn't really needed for `vite`. I've used `rimraf` in the past, which works but introduces an annoying catch 22 where you can't clean without having your dev environment set up. Maybe that's ok though, especially if we can install `buf` via `npm`

Reply:
I gave Windows support a little timebox, but the issues seem a little larger than just `rm` support. Punting on it for now, will file a ticket to address at a later date

Reply:
agreed, we can tackle this later